src/jit/x64/arch.c

```diff
@@ -0,0 +1,103 @@
+#include "moar.h"
+#include "jit/internal.h"
+
+#define MVM_JIT_PLATFORM_POSIX 1
+#define MVM_JIT_PLATFORM_WIN32 2
+
+#ifndef MIN
+#define MIN(a,b) ((a) < (b) ? (a) : (b))
+#endif
```

Comment:
I'm sure I saw this defined in `internal.h`

Reply:
Correct, and then also undefined, because I didn't want them to leak out, i.e. to have nonsymbolic values... I'm not sure about the legitimacy of that concern